conf/app.routes

```diff
@@ -250,3 +250,24 @@ GET        /totalIncome/:period/:schemeIndex                  controllers.annual
+POST       /changeContributedToDuringRemedyPeriod/:period/:schemeIndex                  controllers.annualallowance.taxyear.ContributedToDuringRemedyPeriodController.onSubmit(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+
+GET        /definedContributionAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.DefinedContributionAmountController.onPageLoad(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+POST       /definedContributionAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.DefinedContributionAmountController.onSubmit(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+GET        /changeDefinedContributionAmount/:period/:schemeIndex            controllers.annualallowance.taxyear.DefinedContributionAmountController.onPageLoad(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+POST       /changeDefinedContributionAmount/:period/:schemeIndex            controllers.annualallowance.taxyear.DefinedContributionAmountController.onSubmit(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+
+GET        /definedBenefitAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.DefinedBenefitAmountController.onPageLoad(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+POST       /definedBenefitAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.DefinedBenefitAmountController.onSubmit(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+GET        /changeDefinedBenefitAmount/:period/:schemeIndex            controllers.annualallowance.taxyear.DefinedBenefitAmountController.onPageLoad(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+POST       /changeDefinedBenefitAmount/:period/:schemeIndex            controllers.annualallowance.taxyear.DefinedBenefitAmountController.onSubmit(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+
+GET        /flexiAccessDefinedContributionAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.FlexiAccessDefinedContributionAmountController.onPageLoad(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+POST       /flexiAccessDefinedContributionAmount/:period/:schemeIndex                  controllers.annualallowance.taxyear.FlexiAccessDefinedContributionAmountController.onSubmit(mode: Mode = NormalMode, period:Period, schemeIndex:SchemeIndex)
+GET        /changeFlexiAccessDefinedContributionAmount/:period/:schemeIndex                        controllers.annualallowance.taxyear.FlexiAccessDefinedContributionAmountController.onPageLoad(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
+POST       /changeFlexiAccessDefinedContributionAmount/:period/:schemeIndex                        controllers.annualallowance.taxyear.FlexiAccessDefinedContributionAmountController.onSubmit(mode: Mode = CheckMode, period:Period, schemeIndex:SchemeIndex)
```

Comment:
These url paths are not consistent with the other kebab cases defined. Are the required url's stated anywhere, or can we align with the prototype, or are you expecting that this is addressed by a subsequent ticket?

Reply:
As discussed, covered in content ticket